metaflow/meta_files.py

```diff
@@ -0,0 +1,132 @@
+import json
+import os
+
+from enum import Enum
+from typing import Any, Dict, Optional, Union
+
+from .util import get_metaflow_root
+
+_info_file_content = None
+_info_file_present = None
+_included_dist_info = None
+_included_dist_present = None
+
+# Ideally these would be in package/mfenv.py but that would cause imports to fail so
+# moving here. The reason is that this is needed to read extension information which needs
+# to happen before mfenv gets packaged.
+
+MFENV_DIR = (
+    ".mfenv"  # Directory containing "system" code (metaflow and user dependencies)
+)
+MFCONF_DIR = ".mfconf"  # Directory containing Metaflow's configuration files
```

Comment:
consider changing the name to something else since it is neither metaflow config not the new config feature

Reply:
So we can have `MFCODE_DIR` and `MFINFO_DIR` ?